index.js

```diff
@@ -24,9 +24,11 @@ module.exports = co['default'] = co.co = co;
  */
 
 co.wrap = function (fn) {
-  return function () {
+  function createPromise() {
     return co.call(this, fn.apply(this, arguments));
-  };
+  }
+  createPromise.__generatorFunction__ = fn;
```

Comment:
It's a semi-public API so I picked something more explicit and also less likely to have conflicts with libraries that add properties to Function.prototype.